### ISSUE TITLE
Include some components in Picnic CSS

### ISSUE THREAD

**Issue by FranciscoP** [LFeh/css-components#28]
I am the creator of [Picnic CSS](http://picnicss.com/) and I've found out about [CSS Components](http://www.felipefialho.com/css-components/) recently... and I love it! (:

After investigating each of them, I find the components `Tab` and `Tooltip` specially useful for including them in Picnic CSS as plugins. I already have a [modal plugin](http://www.picnicss.com/plugins/modal/) in my library, otherwise I'd include it too.

From your MIT license I will adapt them for my library needs (not so many classes, SASS), however I wanted to let you know that I'll base these plugins in your great set of Pure CSS components as ir shares a lot in common with Picnic CSS ideas.

**Comment by LFeh** [comment]
Hey man,

Thank you for contacting us. I like your project.

Can use and readjust as needed. ;)

**Comment by FranciscoP** [comment]
You are welcome. I just finished them. I have decided to make the [tabs plugin](http://www.picnicss.com/plugins/tabs) less prescriptive than your version, taking up only 9 LOC and allowing to set up even an [image gallery](http://www.picnicss.com/plugins/tabs/#gallery-example) with it. The disadvantage is more work for the developer to adapt it for each case.

About the [Tooltip Plugin](http://www.picnicss.com/plugins/tooltip/), the main difference is that there's a default one (bottom) and that I rely in the presence of `[data-tooltip]` css selector to show it or not instead of relying in a class. Just my preference.

Again, thanks for the library (:
